mcs/class/Makefile

```diff
@@ -50,6 +50,7 @@ endif
 
 mobile_common_dirs := \
 	corlib	\
+	Mono.Profiler.Log	\
```

Comment:
Yes. This library will, in the future, allow interacting with the profiler while it's running, so having it available on mobile is important.